ghost/core/test/e2e-frontend/members.test.js

```diff
@@ -488,6 +473,43 @@ describe('Front-end members behavior', function () {
                     });
             });
 
+            it('can fetch member data', async function () {
+                const res = await request.get('/members/api/member')
+                    .expect(200);
+
+                const memberData = res.body;
+                should.exist(memberData);
+
+                // @NOTE: this should be a snapshot test not code
+                memberData.should.have.properties([
+                    'uuid',
+                    'email',
+                    'name',
+                    'firstname',
+                    'expertise',
+                    'avatar_image',
+                    'subscribed',
+                    'subscriptions',
+                    'paid',
+                    'created_at',
+                    'enable_comment_notifications',
+                    'newsletters',
+                    'email_suppression'
+                ]);
+                Object.keys(memberData).should.have.length(13);
+                memberData.should.not.have.property('id');
+                memberData.newsletters.should.have.length(1);
+
+                // @NOTE: this should be a snapshot test not code
+                Object.keys(memberData.newsletters[0]).should.have.length(4);
+                memberData.newsletters[0].should.have.properties([
+                    'id',
+                    'name',
+                    'description',
+                    'sort_order'
```

Comment:
Should the frontend include `sort_order`?

Reply:
yup, needed for the frontend [here](https://github.com/TryGhost/Ghost/blob/2554c0df317f84a33238343a4ad78e46b8a307db/apps/portal/src/utils/helpers.js#L519). not sure why, but that is out of scope for what we try to achieve here.